functions/Get-DbaDbBackupHistory.ps1

```diff
@@ -158,7 +158,7 @@ function Get-DbaDbBackupHistory {
         [switch]$IncludeCopyOnly,
         [Parameter(ParameterSetName = "NoLast")]
         [switch]$Force,
-        [DateTime]$Since = (Get-Date '01/01/1970'),
+        [object]$Since = (Get-Date '01/01/1970'),
```

Comment:
```suggestion
        [psobject]$Since = (Get-Date '01/01/1970'),
```

I don't know the difference, exactly, but `psobject` was recommended to me over `object` and I found it worked better than object.